src/main/java/codewars/seven/NthSmallestElement.java

```diff
@@ -0,0 +1,13 @@
+package codewars.seven;
+
+import java.util.Arrays;
+
+/*
+https://www.codewars.com/kata/5a512f6a80eba857280000fc/train/java
+ */
+public class NthSmallestElement {
+
+    public static int nthSmallest(final int[] arr, final int n) {
+        return Arrays.stream(arr).sorted().toArray()[n-1];
```

Comment:
I see that you use streams here. As a suggestion, you could use `IntStream` here with `.skip()` and `limit()` methods.